### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by mani9793**
Done desired changes in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,13 +12,15 @@
...
+Load `@octokit/plugin-rest-endpoint-methods` and [`@octokit/core`](https://github.com/octokit/core.js) (or core-compatible module) directly from [cdn.skypack.dev](https://cdn.skypack.dev)
+ 
 ```html
 <script type="module">
-  import { Octokit } from "https://cdn.pika.dev/@octokit/core";
-  import { restEndpointMethods } from "https://cdn.pika.dev/@octokit/plugin-rest-endpoint-methods";
+  
+import { Octokit } from "https://cdn.skypack.dev/@octokit/core";
+import { restEndpointMethods } from "https://cdn.skypack.dev/@octokit/plugin-rest-endpoint-methods";
+
```
Comment (gr2m) [inline README.md:23]: The changes look good, but you changed the formatting, could you revert these changes? 

- on line 15 & 17 you added spaces
- you added new empty lines 20 & 23
- you removed the spaces before the `import` statements on line 20/21

we call these "whitespace" changes. Could you revert these, please? Let me know if you have any questions

**Review by gr2m — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mani9793) [reply README.md:23]: Okay, I am creating a new pull request with the above changes .

Comment (gr2m) [reply README.md:23]: No need to create a new pull request, you can make the changes in your existing pull requests, directly in the browser. Open the [files changed](https://github.com/octokit/plugin-rest-endpoint-methods.js/pull/265/files) tab, click on the 3 dots on the top right and then on "edit this file" 

image

You can make the changes right in there and commit your changes when you are done

Comment (mani9793) [reply README.md:23]: Actually, I am having some problem to edit in this pull request as it is requesting me to create a new branch for committing the changes, that's why I have created a new PR.

Comment (gr2m) [reply README.md:23]: Ah I see. The problem might be that GitHub cannot find the fork it created when you edited the file in the first place. That's why it says "unknown repository" 

image

Let me close this pull request and you try again, ok?

Comment (mani9793) [reply README.md:23]: Yes, sure !

**Comment by mani9793** [comment]
I have merged a branch with changes, you can check now.

**Comment by gr2m** [comment]
Changes look good 👍🏼 But you need to create the pull request from your fork to this repository at https://github.com/octokit/plugin-rest-endpoint-methods.js/compare/master...mani9793:master.

**Comment by mani9793** [comment]
I have done from that branch but there is no label thing in this case.

**Comment by gr2m** [comment]
I added the labels, you are all set 👍🏼 

If you'd like to another contribution, we have some [`help wanted` issues](https://github.com/issues?q=is%3Aopen+is%3Aissue+org%3Aoctokit+label%3A%22help%20wanted%22). They are not as straight forward as the `first-timers-only` issues reserved for first-time contributors, but you gotta challenge yourself if you want to grow :)
